test/dlc_tests/ec2/mxnet/inference/test_mxnet_inference.py

```diff
@@ -32,6 +33,18 @@ def test_ec2_mxnet_squeezenet_inference_cpu(mxnet_inference, ec2_connection, reg
     run_ec2_mxnet_inference(mxnet_inference, SQUEEZENET_MODEL, "squeezenet", ec2_connection, "cpu", region, 80, 8081)
 
 
+@pytest.mark.parametrize("ec2_instance_type", MX_EC2_CPU_INSTANCE_TYPE, indirect=True)
+@pytest.mark.parametrize("ei_accelerator_type", ["eia1.large"], indirect=True)
```

Comment:
In this PR, we are testing with eia1.large on PR context. But when EI changes are merged for codebuild, I will change the code to use separate accelerator for separate instance.